web_src/css/base.css

```diff
@@ -200,6 +200,9 @@ a.label,
 
 .ui.search > .results .result {
   background: var(--color-body);
+  border-color: var(--color-secondary);
+  display: flex;
+  align-items: center;
```

Comment:
Are there other `.ui.search` components? Does this changed layout work with all of them?

Reply:
Yes there are more, but likely it won't break them. Should test them though.

Reply:
Actually maybe this is the only cas where there are `.results`.